### PR TITLE
Feat frontend via templates, using SQLModel and HTMX for interaction

### LINKED ISSUE #9: Integer OverFlow at SQLite database access
Input validation missing.

```
INFO:     10.75.0.220:56048 - "POST /messages HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/uvicorn/protocols/http/httptools_impl.py", line 419, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/applications.py", line 123, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/middleware/errors.py", line 186, in __call__
    raise exc
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/middleware/errors.py", line 164, in __call__
    await self.app(scope, receive, _send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/middleware/cors.py", line 91, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/middleware/cors.py", line 146, in simple_response
    await self.app(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/routing.py", line 758, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/routing.py", line 778, in app
    await route.handle(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/routing.py", line 299, in handle
    await self.app(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/routing.py", line 79, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 64, in wrapped_app
    raise exc
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    await app(scope, receive, sender)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/starlette/routing.py", line 74, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/fastapi/routing.py", line 299, in app
    raise e
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/fastapi/routing.py", line 294, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/fastapi/routing.py", line 191, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/main.py", line 155, in send_message
    session.commit()
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1972, in commit
    trans.commit(_to_root=True)
  File "<string>", line 2, in commit
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go
    ret_value = fn(self, *arg, **kw)
                ^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1257, in commit
    self._prepare_impl()
  File "<string>", line 2, in _prepare_impl
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go
    ret_value = fn(self, *arg, **kw)
                ^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1232, in _prepare_impl
    self.session.flush()
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 4296, in flush
    self._flush(objects)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 4431, in _flush
    with util.safe_reraise():
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__
    raise exc_value.with_traceback(exc_tb)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 4392, in _flush
    flush_context.execute()
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/unitofwork.py", line 466, in execute
    rec.execute(self)
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/unitofwork.py", line 642, in execute
    util.preloaded.orm_persistence.save_obj(
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/persistence.py", line 93, in save_obj
    _emit_insert_statements(
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/persistence.py", line 1233, in _emit_insert_statements
    result = connection.execute(
             ^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1408, in execute
    return meth(
           ^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/sql/elements.py", line 513, in _execute_on_connection
    return connection._execute_clauseelement(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1630, in _execute_clauseelement
    ret = self._execute_context(
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1839, in _execute_context
    return self._exec_single_context(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1979, in _exec_single_context
    self._handle_dbapi_exception(
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 2338, in _handle_dbapi_exception
    raise exc_info[1].with_traceback(exc_info[2])
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1960, in _exec_single_context
    self.dialect.do_execute(
  File "/opt/kiezbox-messages-backend/backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 924, in do_execute
    cursor.execute(statement, parameters)
OverflowError: Python int too large to convert to SQLite INTEGER
```

### PR DESCRIPTION
**Pull request by MRigal**

### CODE REVIEW

**Comment by MRigal** [comment]
@ZenVega bin noch dran, auch funktionell noch nicht funktionierend, aber du könntest ein Gefühl dafür haben, was noch UI/Frontend mäßig zu tun ist.

**Comment by ZenVega** [comment]
Hab das styling der Inpbox noch ein bisschen aufgeräumt

**Comment by james-knippes** [comment]
#9 may have been introduced by 1c2e419bd8eff3042bc66fb7ff691e788f52d72b ?

**Comment by MRigal** [comment]
It was always in. But I will add additional frontend validation

> #9 may have been introduced by [1c2e419](https://github.com/technologiestiftung/kiezbox-messages-backend/commit/1c2e419bd8eff3042bc66fb7ff691e788f52d72b) ?

File: backend/main.py [hunk 1]
```diff
@@ -26,4 +35,165 @@
...
+    SQLModel.metadata.create_all(engine)
+
+
+@app.on_event("startup")
+def on_startup():
+    init_db()
+
+
+# Messages event generator
+STREAM_DELAY = 5  # In seconds
```
Comment (guadiromero) [inline backend/main.py]: @MRigal we could turn this into an env variable, no?

**Review by guadiromero — APPROVED** [review]
@MRigal Looks good to me, but I would suggest that we still meet with you and Urs so you can explain to us an overview of the changes. From my part, especially what goes on on `main.py`

I haven't taken a look at the JS code, I leave that to @ZenVega :)

File: backend/main.py (hunk 1 above)
Comment (MRigal) [reply backend/main.py]: I can make it conditional, but there should be a default, as it is not something we're not gonna change frequently. Also adding env variables to the os custom builds adds chances that it gets lost somewhere on the way, it's not a managed service with easy GUI remote config...

**Pull request merged**